### ISSUE TITLE
For objects

### ISSUE THREAD

**Issue by Dratlan** [zjumsj/GaussianBlendshapes#23]
If I prepare facial features for objects(like animals, cubes, flowers,etc.) reconstructed by Gaussian, how should i get their gaussianblendshapes in your work?

**Comment by zjumsj** [comment]
That won't be easy. In our work, "Gaussian blendshapes" are not directly extractable from a generic static 3DGS reconstruction—you need training data with sufficient expression diversity to train a generalizable model.

If you're trying to (1) obtain blendshapes from static 3DGS, or (2) reconstruct dynamics from an expression video (e.g., of a cat). For case (2), you would need a cat-specific mesh blendshape model and a cat-specific tracker to estimate expression coefficients (which are likely uncommon ?).

For case (1), a more practical approach might be to manually annotate sparse landmarks to establish correspondence between the GS model and a 3DMM (e.g., FLAME), then extract deformation gradients to deform the Gaussian model. However, this is essentially deformation transfer/retargeting, which goes beyond the scope of "Gaussian blendshapes."

**Comment by Dratlan** [comment]
Thanks for your detailed reply. I'll take it into consideration and give it a try.
